stream_file_builder.go

```diff
@@ -119,23 +125,83 @@ func (sb *StreamFileBuilder) AddSheet(name string, headers []string, cellTypes [
 	return nil
 }
 
+// TODO update comments
+// AddSheetWithStyle will add sheets with the given name with the provided headers. The headers cannot be edited later, and all
+// rows written to the sheet must contain the same number of cells as the header. Sheet names must be unique, or an
+// error will be thrown. Additionally AddSheetWithStyle allows to add Style information to the headers.
+func (sb *StreamFileBuilder) AddSheetWithStyle(name string, cells []StreamCell) error {
```

Comment:
I would rename this function to `AddSheetS` and refactor it to have the following signature:

```go
func (sb *StreamFileBuilder) AddSheetS(name string, nbColumns int) error
```

Reply:
I changed this to:
`func (sb *StreamFileBuilder) AddSheetS(name string, columnStyles []StreamStyle) error`
So that default column styles can be added.